crates/bevy_reflect/src/impls/std.rs

```diff
@@ -298,3 +305,113 @@ impl GetTypeRegistration for Cow<'static, str> {
         registration
     }
 }
+
+impl<T: Reflect, const N: usize> Array for [T; N] {
```

Comment:
Can you implement it for `[T]` too? (slices)

Reply:
Hmm I don't think this is possible since `dyn Any` requires the type to be sized and `[T]` is not sized :(